code/datums/ai_laws.dm

```diff
@@ -318,14 +318,20 @@
 /datum/ai_laws/proc/add_inherent_law(law)
 	if (!(law in inherent))
 		inherent += law
+		owner?.lawcheck += law
```

Comment:
in what situation would owner ever be missing

Reply:
same goes everywhere else

Reply:
I had it because it runtimed during testing.. however it seems to not happen anymore. I'll remove it

Reply:
nvm its because of MMIs

Reply:
why because of MMI's?

this seems like a bug

Reply:
it is, ill fix it

Reply:
anyways, MMIs and empty AI cores get the ai_laws datum, so thats when owner is null